language/de/install.php

```diff
@@ -632,3 +259,155 @@
 
 	'TOPICS_TOPIC_TITLE'			=> 'Willkommen bei phpBB3!',
 ));
+
+// Common navigation items' translation
+$lang = array_merge($lang, array(
+	'MENU_OVERVIEW'		=> 'Übersicht',
+	'MENU_INTRO'		=> 'Einführung',
+	'MENU_LICENSE'		=> 'Lizenz',
+	'MENU_SUPPORT'		=> 'Support',
+));
+
+// Task names
+$lang = array_merge($lang, array(
+	// Install filesystem
+	'TASK_CREATE_CONFIG_FILE'	=> 'Erstelle Konfigurationsdatei',
+
+	// Install database
+	'TASK_ADD_CONFIG_SETTINGS'		=> 'Füge Konfigurationsteinstellungen hinzu',
+	'TASK_ADD_DEFAULT_DATA'			=> 'Lege Standard-Einstellungen in der Datenbank fest',
+	'TASK_CREATE_DATABASE_SCHEMA'	=> 'Erstelle Datenbankstruktur',
+
+	// Install data
+	'TASK_ADD_BOTS'			=> 'Registriere Bots',
+	'TASK_ADD_LANGUAGES'	=> 'Installiere verfügbare Sprachen',
+	'TASK_ADD_MODULES'		=> 'Installiere Module',
+
+	// Install finish tasks
+	'TASK_NOTIFY_USER'			=> 'Sende Benachrichtigungs-Mail',
+	'TASK_POPULATE_MIGRATIONS'	=> 'Rolle Migrationen aus', // TODO: passende Übersetzung
```

Comment:
In der migrator.php haben wir mehrfach "durchführen" verwendet - das würde ich dann auch hier anwenden